src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -128,7 +128,7 @@
   <ItemGroup>
     <PackageReference Include="DnsClient" Version="1.3.1" />
     <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.2" PrivateAssets="All" />
-    <PackageReference Include="MongoDB.LibMongocrypt" Version="202.202.202" />
+    <PackageReference Include="MongoDB.LibMongocrypt" Version="300.0.0" />
```

Comment:
What is version `300`?

Reply:
just random value, this should be the same version as in the linux PR, I just forgot to update it. Will do it